src/V3FileLine.h

```diff
@@ -265,6 +265,19 @@ class FileLine {
                 && m_lastLineno == rhs.m_lastLineno && m_lastColumn == rhs.m_lastColumn
                 && m_filenameno == rhs.m_filenameno && m_warnOn == rhs.m_warnOn);
     }
+    // Returns -1 if (*this) should come before rhs after sorted. 1 for the opposite case. 0 for
+    // equivalent.
+    int operatorCompare(const FileLine& rhs) const {
+        if (m_filenameno != rhs.m_filenameno) return m_filenameno < rhs.m_filenameno ? -1 : 1;
```

Comment:
```suggestion
        if (m_filenameno != rhs.m_filenameno) return (m_filenameno < rhs.m_filenameno) ? -1 : 1;
```
Ditto elsewhere (style tools complain otherwise).

Reply:
Add parenthesis in cbd1823